xbmc/input/ButtonTranslator.cpp

```diff
@@ -1155,9 +1155,8 @@ int CButtonTranslator::GetFallbackWindow(int windowID)
     if (fallbackWindows[index].origin == windowID)
       return fallbackWindows[index].target;
   }
-  // for addon windows use WINDOW_ADDON_START
-  // because id is dynamic
-  if (windowID >= WINDOW_ADDON_START && windowID <= WINDOW_ADDON_END)
+  // for addon windows use WINDOW_ADDON_START because id is dynamic
+  if (windowID > WINDOW_ADDON_START && windowID <= WINDOW_ADDON_END)
```

Comment:
Why do you think we need to adjust the initialization? `GetFallbackWindow()` is called in case there is no mapping found for the given window. The given window is always WINDOW_ADDON_START + X and if there is no mapping found for WINDOW_ADDON_START + X it calls `GetFallbackWindow()` and return WINDOW_ADDON_START. If there is not mapping found for WINDOW_ADDON_START it'll fallback to -1 default section.